docs/templates/eap73-openjdk11-openjdk11-amq-persistent-s2i.adoc

```diff
@@ -0,0 +1,387 @@
+////
+    AUTOGENERATED FILE - this file was generated via ./gen_template_docs.py.
+    Changes to .adoc or HTML files may be overwritten! Please change the
+    generator or the input template (./*.in)
+////
+
+= eap73-amq-persistent-s2i
```

Comment:
* why does `openjdk11` appear twice in the file name?
* shouldn't the title of the file match its filename?

Reply:
I'll check the docs generation script, as far as I know we haven't changed anything there, but it does prefix the image name into the file. The filename should match though, yes.

Reply:
This was an error, I've removed the bad doc files and regenerated the adoc files, so this should be ok now.